editor/src/components/canvas/controls/select-mode/simple-outline-control.tsx

```diff
@@ -1,20 +1,45 @@
 import React from 'react'
+import * as EP from '../../../../core/shared/element-path'
 import { ElementPath } from '../../../../core/shared/project-file-types'
+import { when } from '../../../../utils/react-conditionals'
 import { useColorTheme } from '../../../../uuiui'
 import { useEditorState } from '../../../editor/store/store-hook'
 import { useBoundingBox } from '../bounding-box-hooks'
 import { getSelectionColor } from '../outline-control'
 
-interface OutlineControlProps {
+interface MultiSelectOutlineControlProps {
   localSelectedElements: Array<ElementPath>
 }
 
-export const OutlineControl = React.memo<OutlineControlProps>((props) => {
-  const colorTheme = useColorTheme()
+export const MultiSelectOutlineControl = React.memo<MultiSelectOutlineControlProps>((props) => {
   const localSelectedElements = props.localSelectedElements
+  return (
+    <>
+      {[
+        <OutlineControl
+          key='multiselect-outline'
+          targets={localSelectedElements}
+          color='multiselect-bounds'
+        />,
+        ...localSelectedElements.map((path) => (
+          <OutlineControl key={EP.toString(path)} targets={[path]} color='primary' />
+        )),
+      ]}
+    </>
+  )
+})
+
+interface OutlineControlProps {
+  targets: Array<ElementPath>
```

Comment:
With the `MultiSelectOutlineControl` does it still make sense to use an array here?

Reply:
I think so! for the outline bounds case